src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+  
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (h) => (
+  <button {...props}>{text}</button>
+);
+
+Button.propTypes = {
+  text: PropTypes.string.isRequired
+};
```

Comment:
it looks like you're passing `h` as props, but your prop types say it's expecting `text` ...?